packages/orbit-components/src/TextLink/index.tsx

```diff
@@ -78,16 +78,23 @@ const getSizeToken = ({ theme, size }: { theme: Theme; size: Common.Size }): str
   return sizeTokens[size];
 };
 
+const getIconSize = (theme: Theme, size: Props["size"]) => {
+  if (size === SIZE_OPTIONS.LARGE)
+    return { width: theme.orbit.widthIconLarge, height: theme.orbit.heightIconLarge };
+  if (size === SIZE_OPTIONS.SMALL)
+    return { width: theme.orbit.widthIconSmall, height: theme.orbit.heightIconSmall };
+  return { width: theme.orbit.widthIconMedium, height: theme.orbit.heightIconMedium };
+};
+
 const StyledIconContainer = styled(({ children, className }) => (
```

Comment:
I know it was not touched, but I think we can remove here this arrow function with span and have only `styled.span` ;)